### ISSUE TITLE
Twitter experience: explain "App activity"

### ISSUE THREAD

**Issue by foucault-dumas** [hestiaAI/hestialabs-experiences#757]
![image](https://user-images.githubusercontent.com/81250365/173581370-852de737-57db-4790-98be-8bc09ea7536d.png)
Do we know more than the name for the App activity (exlusion/inclusion)? This should be reflected in the interface.

**Comment by pdehaye** [comment]
(also for lists?)

**Comment by fquellec** [comment]
I'm not sure I understand the question, 'App activity' is a targeting criteria, it is associated with a targeting value on the right hand graph. So, for example here, you were targeted because you installed Gorillas Grocery and because you made a purchase on that app.

**Comment by fquellec** [comment]
Yes, these are the targeting criteria used by the advertiser to propagate the corresponding ad, so this means that you match at least one criteria, not all! It's actually more complicated than that, there are different types of targeting criteria, some are mandatory and some are not.

<img width="1001" alt="image" src="https://user-images.githubusercontent.com/17878016/174039382-ac077ea3-99b1-4d78-91af-8667c2377633.png">
[source](https://developer.twitter.com/en/docs/twitter-ads-api/campaign-management/overview/targeting)

**Comment by alexbfree** [comment]
move to backlog
